init.c: mismatched printf format specifiers and arguments

```
init.c:366:52: warning: format specifies type 'char *' but the argument has type 'size_t'
      (aka 'unsigned long') [-Wformat]
            printf("SMA_CMA_PUT_MAX         %s\n", shmem_transport_cma_put_max);
                                            ~~     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                                            %zu
init.c:367:52: warning: format specifies type 'char *' but the argument has type 'size_t'
      (aka 'unsigned long') [-Wformat]
            printf("SMA_CMA_GET_MAX         %s\n", shmem_transport_cma_get_max);
                                            ~~     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                                            %zu
```

--- a/src/init.c
+++ b/src/init.c
@@ -344,8 +344,8 @@ shmem_internal_init(int tl_requested, int *tl_provided)
             printf("SMA_FCOLLECT_ALGORITHM  %s\n", coll_type_str[shmem_internal_fcollect_type]);
             printf("\tAlgorithm for fcollect.  Options are auto, linear, ring, recdbl\n");
 #ifdef USE_CMA
-            printf("SMA_CMA_PUT_MAX         %s\n", shmem_transport_cma_put_max);
-            printf("SMA_CMA_GET_MAX         %s\n", shmem_transport_cma_get_max);
+            printf("SMA_CMA_PUT_MAX         %zu\n", shmem_transport_cma_put_max);
+            printf("SMA_CMA_GET_MAX         %zu\n", shmem_transport_cma_get_max);
 #endif /* USE_CMA */
             printf("SMA_DEBUG               %s\n", (NULL != shmem_util_getenv_str("DEBUG")) ? "On" : "Off");
             printf("\tEnable debugging messages.\n");

--- a/src/transport_ofi.c
+++ b/src/transport_ofi.c
@@ -443,7 +443,6 @@ static inline int allocate_recv_cntr_mr(void)
 {
 
     int ret = 0;
-    struct fi_cntr_attr cntr_attr = {0};
 
     /* ------------------------------------*/
     /* POST enable resources for to EP     */
@@ -451,16 +450,20 @@ static inline int allocate_recv_cntr_mr(void)
     /* since this is AFTER enable and RMA you must create memory regions for incoming reads/writes
      * and outgoing non-blocking Puts, specifying entire VA range */
 
-    // Create counter for incoming writes
-    cntr_attr.events   = FI_CNTR_EVENTS_COMP;
-    cntr_attr.flags    = 0;
-
 #ifndef ENABLE_HARD_POLLING
-    ret = fi_cntr_open(shmem_transport_ofi_domainfd, &cntr_attr,
-                       &shmem_transport_ofi_target_cntrfd, NULL);
-    if(ret!=0){
-        OFI_ERRMSG("target cntr_open failed\n");
-        return ret;
+    {
+        struct fi_cntr_attr cntr_attr = {0};
+
+        // Create counter for incoming writes
+        cntr_attr.events   = FI_CNTR_EVENTS_COMP;
+        cntr_attr.flags    = 0;
+
+        ret = fi_cntr_open(shmem_transport_ofi_domainfd, &cntr_attr,
+                           &shmem_transport_ofi_target_cntrfd, NULL);
+        if(ret!=0){
+            OFI_ERRMSG("target cntr_open failed\n");
+            return ret;
+        }
     }
 #endif
 

